library/gcm.c

```diff
@@ -194,78 +299,79 @@ static const uint16_t last4[16] =
     0x9180, 0x8da0, 0xa9c0, 0xb5e0
 };
 
-/*
- * Sets output to x times H using the precomputed tables.
- * x and output are seen as elements of GF(2^128) as in [MGV].
- */
-static void gcm_mult(mbedtls_gcm_context *ctx, const unsigned char x[16],
-                     unsigned char output[16])
+static void gcm_mult_smalltable(uint8_t *output, const uint8_t *x, uint64_t H[16][2])
 {
     int i = 0;
     unsigned char lo, hi, rem;
-    uint64_t zh, zl;
+    uint64_t u64z[2];
+    const uint64_t *pu64z = 0;
```

Comment:
```suggestion
    const uint64_t *pu64z = NULL;
```

Please use NULL instead of 0 to initialise the pointer.